addcustomer.php

```diff
@@ -11,6 +11,7 @@
 ?>
 <!DOCTYPE html>
 <html lang="en">
+  
```

Comment:
Take out all frontend implementations. I'll like to see a queue system implemented from scratch. You can use a package, but use the package to do something with the backend (no need for front end). 

For example, show what happens after a queue is created, stored (in DB), and dispatched. The following questions need to be answered:

1. How does the system keep track of already dispatched queues
2. How does the system keep track of completed queues
3. If a queue files, how does the system know, and what happens?
4. Would we need a cron job to get the entire thing working?